frontend/src/components/SettingsConnections.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="flex flex-col gap-8">
+    <div class="text-3xl font-thin text-gray-500 dark:text-gray-200">{{ t('heading.connectedAccounts') }}</div>
+    <div class="max-w-3xl pl-6" v-for="(connection, category) in connections" v-bind:key="category">
+      <h2 class="mb-4 text-xl font-medium">{{ t(`heading.settings.connectedAccounts.${category}`) }}</h2>
+      <p>{{ t(`text.settings.connectedAccounts.connect.${category}`) }}</p>
+      <div v-if="category === 'google'" class="pt-2">
+        <p>
+          <i18n-t :keypath="`text.settings.connectedAccounts.connect.${category}Legal.text`" tag="label" :for="`text.settings.connectedAccounts.connect.${category}Legal.link`">
+          <a class="underline" href="https://developers.google.com/terms/api-services-user-data-policy" target="_blank">{{ t(`text.settings.connectedAccounts.connect.${category}Legal.link`) }}</a>
+          </i18n-t>
+        </p>
+      </div>
+      <div class="mt-4 flex items-center pl-4">
+        <div class="w-full max-w-md">
+          <p v-if="connection[0]">{{ t('label.connectedAs', { name: connection[0].name }) }}</p>
+          <p v-if="!connection[0]">{{ t('label.notConnected') }}</p>
+        </div>
+        <div class="mx-auto mr-0" v-if="category !== 'fxa'">
+          <primary-button
+          v-if="!connection[0]"
+          :label="t('label.connect')"
+          @click="() => connectAccount(category)"
+        />
+        <caution-button
+          v-if="connection[0]"
+          :label="t('label.disconnect')"
+          @click="() => displayModal(category)"
+        />
+        </div>
+        <div class="mx-auto mr-0" v-else>
+          <secondary-button
+          :label="t('label.editProfile')"
+          @click="editProfile"
+          />
+        </div>
+      </div>
+    </div>
+  </div>
+  <!-- Disconnect Google Modal -->
+  <ConfirmationModal
+      :open="disconnectGoogleModalOpen"
+      :title="t('text.settings.connectedAccounts.disconnect.google.title')"
+      :message="t('text.settings.connectedAccounts.disconnect.google.message')"
+      :confirm-label="t('text.settings.connectedAccounts.disconnect.google.confirm')"
+      :cancel-label="t('text.settings.connectedAccounts.disconnect.google.cancel')"
+      :use-caution-button="true"
+      @confirm="() => disconnectAccount('google')"
+      @close="closeModals"
+  ></ConfirmationModal>
+  <!-- Disconnect Zoom Modal -->
+  <ConfirmationModal
```

Comment:
todo (non-blocking): dash-case (see above)

```suggestion
  <confirmation-modal
```